src/victorops.coffee

```diff
@@ -187,6 +187,8 @@ class VictorOps extends Adapter
       for item in data.PAYLOAD.TIMELINE_LIST
         try
           if item.ALERT
+            @robot.brain.set item.ALERT["INCIDENT_NAME"], item.ALERT
```

Comment:
Does this lead to an ever-growing key-value store?  It's not clear to me if this key-value store survives hubot restarts.  Is this a concern?